blog/2019-10-02-MigratingOptaWebEmployeeRosteringToSpringBoot.adoc

```diff
@@ -0,0 +1,204 @@
+= What's New in OptaWeb Employee Rostering? (part 2) - Making the Backend Cloud-Ready
+:page-interpolate: true
+:awestruct-author: jucui
+:awestruct-layout: blogPostBase
+:awestruct-tags: [employee rostering, coding, algorithm, production]
+
+Recently, the OptaPlanner team has been shifting towards cloud-native applications.
+As a part of this effort, we migrated the OptaWeb Employee Rostering Backend Application from WildFly to Spring Boot.
+
+OptaWeb Employee Rostering is a web application that optimizes shift scheduling for maximum efficiency and fairness for employees.
+In this post we’ll give an overview of the new Spring backend application, and the key features that were added on top of the existing backend.
+
+== The Problem
+
+Let’s start with the problem we’re trying to solve.
+Here is a visual overview of the https://www.optaplanner.org/learn/useCases/employeeRostering.html[Employee Rostering problem]:
+
+image::employeeRosteringProblem.png[]
```

Comment:
It's very good to use many images.
It's good to reuse images from the docs.
It's best to indeed to copy them here, practically speaking.

Can you keep the same file name though if they are unaltered? Do give them a different name if they altered.
If we update those employeeRosteringValueProposition.png in the docs, it might worth updating this blog post too, in a few years down the line. (Blog post can live very long).